### PR TITLE
feat(colors): add color palette - INNO-299

### PR DESCRIPTION
**Pull request by yhuard**
Add a color palette.

palette

TODO:

- [ ] Add meaning to the colors
- [ ] Classify the colors by meaning

### CODE REVIEW

**Comment by degliwe** [comment]
at the moment it is simply a way to display a palette. this isn't related to scss.
Good first step, I would keep it in the feature branch until we make it functional (ex. display the palette from the scss or inject the palette into scss...)

**Comment by emeryro** [comment]
Seems good to me, but as Wesley said, we should link it to sass variables.
In the styleguide I made for PE, I added some helpers classes for each color (named bg-color__primary and text-color__primary for instance) to quickly use color palette in all other components.
That was quite useful...

**Comment by emeryro** [comment]
:+1:

File: framework/_partials/_preview-palettes.twig [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    <link rel="stylesheet" href="{{ '/framework/styles/europa.css' | path }}">
+    <title>Preview Layout</title>
+    <style>
+    body {
+      background-color: #f0f2f5;
+    }
+    .palette {
+        border-radius: 5px;
+        background-color: #fff;
+        -webkit-box-shadow: 0px 3px 0px 0px rgba(226, 228, 231, 0.75);
```
Comment (kalinchernev) [inline framework/_partials/_preview-palettes.twig]: [Not required](http://shouldiprefix.com/#box-shadow) actually.

**Comment by kalinchernev** [comment]
I was just in the FFAUA file for [color palettes](https://docs.google.com/drawings/d/1xH1ekWE7d9qJ18jlWYvnokpYdv5aJGdkippD39BsUeQ/edit) so quickly updated this to latest master.
Couldn't manage to find the POC for the js to scss integration @yhuard experiments though.

File: framework/_partials/_preview-palettes.twig (hunk 1 above)
Comment (yhuard) [inline framework/_partials/_preview-palettes.twig]: My bad, I only copied/pasted this part 👎

**Comment by nelsongrin** [comment]
HI guys,
I'm adding colors and  renaming the colo names that we have based on the design team requirements.
Please put this on hold.
Thanks

**Pull request closed without merging**